tools/client/client.go

```diff
@@ -0,0 +1,115 @@
+package client
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/threefoldtech/zos/pkg/capacity"
+	"github.com/threefoldtech/zos/pkg/capacity/dmi"
+	"github.com/threefoldtech/zos/pkg/schema"
+	"github.com/threefoldtech/zos/tools/bcdb_mock/models/generated/directory"
+	"github.com/threefoldtech/zos/tools/bcdb_mock/models/generated/phonebook"
+	"github.com/threefoldtech/zos/tools/bcdb_mock/models/generated/workloads"
+)
+
+// Client structure
+type Client struct {
```

Comment:
I thought of that, but I thought may be it's better if we have the client per mock "pkg" so directory, phonebook and workloads, otherwise I will have to always pass around 2 (or more) insntance of the interfaces 
for example, modules that need to work against reservations will usually need workloads and reservations, etc ... so it's cleaner to have related stuff grouped in one interface